packages/ui/src/AppSidebar/AppSidebar.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactNode, useCallback, useState } from 'react'
+import cn from 'classnames'
+import { ChevronsLeft, ChevronsRight } from 'react-feather'
+import createPersistedState from 'use-persisted-state'
+
+import { IconButton } from '../IconButton'
+import { useOpenCloseState } from '../hooks'
+import { AppSidebarFavorites } from './AppSidebarFavorites'
+import { appSidebarContext, FavoriteItem } from './appSidebarContext'
+
+import styles from './AppSidebar.module.scss'
+
+export interface AppSidebarProps {
+  children?: ReactNode
+  className?: string
+}
+
+export const AppSidebar = ({ children, className }: AppSidebarProps) => {
+  const { isOpen, toggle } = useOpenCloseState(true)
```

Comment:
We were also keeping this as a persisted state, as far as i remember.

Reply:
done